specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2022-10-01-preview/ResourceHealth.json

```diff
@@ -0,0 +1,1979 @@
+          "ImpactedResourcesGet": {
+            "$ref": "./examples/ImpactedResources_Get.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ResourceHealth/events/{eventTrackingId}/impactedResources": {
+      "get": {
+        "tags": [
+          "EventImpactedResources"
+        ],
+        "operationId": "ImpactedResources_ListByTenantIdAndEventId",
+        "description": "Lists impacted resources in the tenant by an event.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/EventTrackingIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/FilterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of impacted resources in the tenant by an event.",
+            "schema": {
+              "$ref": "#/definitions/eventImpactedResourceListResult"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ListEventsByTenantId": {
+            "$ref": "./examples/ImpactedResources_ListByTenantId_ListByEventId.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ResourceHealth/events/{eventTrackingId}/impactedResources/{impactedResourceName}": {
+      "get": {
+        "tags": [
+          "EventImpactedResources"
+        ],
+        "operationId": "ImpactedResources_GetByTenantId",
+        "description": "Gets the specific impacted resource in the tenant by an event.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/EventTrackingIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ImpactedResourceNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The details of Impacted Resource.",
+            "schema": {
+              "$ref": "#/definitions/eventImpactedResource"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ImpactedResourcesGet": {
+            "$ref": "./examples/ImpactedResources_GetByTenantId.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ResourceHealth/events/{eventTrackingId}/listSecurityAdvisoryImpactedResources": {
+      "post": {
```

Comment:
Why use "POST" instead of GET at multiple resources.

Reply:
This URL path should have RBAC controls. Subscription Reader roles can access GET API. We want users with only certain roles (more restrictive than Subscription/Tenant reader) to access data from this API.

Here is the stack overflow question, we asked https://stackoverflow.microsoft.com/questions/328094

Reply:
Here is the merge PR for the relevant commit - https://github.com/Azure/azure-rest-api-specs/pull/21758